src/components/App.js

```diff
@@ -1,14 +1,41 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPannel from './ButtonPannel';
-// eslint-disable-next-line no-unused-vars
 import calculate from '../logic/calculate';
 
-const App = () => (
-  <div className="app m-auto">
-    <Display />
-    <ButtonPannel />
-  </div>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
- [ ] If you write the handleClick function below using arrow functions you won't need to write the binding inside the constructor.

[A cool article about it here.](https://medium.com/@joespinelli_6190/using-arrow-functions-to-avoid-binding-this-in-react-5d7402eec64)